src/auth.go

```diff
@@ -38,10 +38,15 @@ func (a *authBase) getCommand() string {
 
 // Performs input selection user. If saving last user is enabled (PerTty/Global), user is read from defined path and used as predefined value.
 func (a *authBase) selectUser(c *config) (string, error) {
+	indent := ""
+	if c.IndentSelection > 0 {
+		indent = strings.Repeat(" ", c.IndentSelection)
```

Comment:
`indent = strings.Repeat(" ", c.IndentSelection)`
This part is repeated on many places. It could be done once in config (e.g. a new variable without assignation to loading from config, where you just place space indentation; in custom positiveIntParser/altrrnative, where you reuse its result and then prepares the indentation).

But that's just nice to have, feel free to ignore this comment.